### ISSUE TITLE
.asBitmap method.

### ISSUE THREAD

**Issue by jzeferino** [beraybentesen/glide-xamarin-android#8]
@beraybentesen the asBitmap method is not visisble.

More users with the same problem.

**Comment by beraybentesen** [comment]
Yes, that is because Xamarin binding mechanism doesn't like abstract classed so I had to remove that node in the project.

**Comment by jzeferino** [comment]
ok, but its a crucial method, like `asGif` for example. `RequestManager ` isn't a [Abstract](https://github.com/bumptech/glide/blob/3761a432c0f58fd77cd9d06314a1c6c17bca35f0/library/src/main/java/com/bumptech/glide/RequestManager.java#L46) class.

**Comment by jzeferino** [comment]
@beraybentesen do you plan to add that methods to the binding?

**Comment by MagneticLlama** [comment]
This is a critical issue for me.

**Comment by beraybentesen** [comment]
Published version 4.1.1, could give a chance ? [NuGet](https://www.nuget.org/packages/Glide.Xamarin/4.1.1)

**Comment by MagneticLlama** [comment]
Updated to 4.1.1, and the .AsBitmap method is not available on RequestManager. Worse yet, a lot of new things seem to be broken. I'll create a new issue for those.

**Comment by jzeferino** [comment]
@MagneticLlama @beraybentesen `.AsBitmap()` is now available in 4.1.1 but as I described in #14 the project need to be targeting Android 7 at least. 

It will be nice if you lower the minimum target version to android 5 or 6.
